### DISCUSSION TITLE
Rename the blacklist option

### DISCUSSION THREAD

**Discussion by jordancrick** [lawand-dot-io/taskbar#17] [Feature requests]
https://www.splunk.com/en_us/blog/learn/blacklist-whitelist-inclusivity.html

See article for details.

**Comment by lawand-dot-io** [comment]
Thanks for mentioning this :) I should implement it at some point. What do you think is the term that users will understand? Always-hide list? I need to think about a term that most users will immediately understand.

**Reply by jordancrick** [reply]
Always Hide is Great. You can put that on the preferences menu and above the list in the preferences put "Always Hidden." I think it will be more clear. Thanks!
